### PR TITLE
Manually merge 4.x into master

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": "^7.2",
-        "friendsofsymfony/ckeditor-bundle": "^1.0",
+        "friendsofsymfony/ckeditor-bundle": "^1.0 || ^2.0",
```
Comment (greg0ire) [inline composer.json:25]: :thinking: when I merged, it looked like we just removed the bundle, but then there still is this dependency… why?

Comment (greg0ire) [reply composer.json:25]: FYI it was removed in #389 @bmaziere

Comment (greg0ire) [reply composer.json:25]: cc @core23  I think I will just remove the dependency.

**Review by jordisala1991 — APPROVED** [review]
Guess the merge is okay... but having such features removed without a path of migration will be hard to deal with

**Pull request merged**